python/ray/tune/tests/test_searchers.py

```diff
@@ -327,9 +327,13 @@ def testSkopt(self):
         self.assertCorrectExperimentOutput(out)
 
     def testZOOpt(self):
+        self.skipTest(
+            "Recent ZOOpt versions fail handling invalid values gracefully. "
+            "Skipping until a fix is added in a future ZOOpt release."
+        )
         from ray.tune.search.zoopt import ZOOptSearch
 
-        np.random.seed(1000)  # At least one nan, inf, -inf and float
+        np.random.seed(1002)  # At least one nan, inf, -inf and float
```

Comment:
OOC why're we changing the seed?

Reply:
The test was passing for me locally with the old seed. I changed the seed and upped the number of samples to get the test failing on my local machine.

Reply:
Ah, okay. Makes sense

Reply:
Does that seed produce one nan, inf, -inf and float? 
Which combination makes it fail?
Let's add this as a comment

Reply:
Yes, this seed also produces at at least one of each. The failure happens when enough training samples have been collected and then a nan shows up and zoopt tries to replace one of its existing solutions with the nan solution. The previous seed only included a nan solution while it was still collecting training data. Will add as a comment!